src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -98,6 +98,15 @@ protected ActionResult execute() {
                                                         new Text(emailContent),
                                                         null);
             data.emailToEdit.emailId = emailId;
+        } else {
+            if (addressModeOn) {
+                statusToAdmin = "Email sent to " + addressReceiverListString;
+                statusToUser.add(new StatusMessage("Email sent to "
```

Comment:
The email is not sent instantly. Should say something like the email will be sent within an hour.